client/web/src/auth/SignInPage.tsx

```diff
@@ -35,16 +40,28 @@ export const SignInPage: React.FunctionComponent<React.PropsWithChildren<SignInP
     const location = useLocation()
     const [error, setError] = useState<Error | null>(null)
 
+    const isOperatorHidingEnabled = props.context.experimentalFeatures.hideSourcegraphOperatorLogin ?? false
+
+    const showSourcegraphOperatorLogin =
+        !isOperatorHidingEnabled || new URLSearchParams(location.search).has('sourcegraph-operator')
+
+    const isSourcegraphOperatorProvider = (provider: AuthProvider): boolean =>
+        provider.serviceType === 'openidconnect' && provider.displayName === 'Sourcegraph Employee'
```

Comment:
Maybe we should take the opportunity to align the naming:

```suggestion
        provider.serviceType === 'openidconnect' && provider.displayName === 'Sourcegraph Operator'
```

Reply:
We'd need to modify all the names configured in instances + we have other relying on the name, so I'd rather keep this as-is and wait until https://docs.google.com/document/d/1Ia9sjW_KQ6BeJ28xJl3VvLIy7M9Ko7A1sBnTXkZ3W9o/edit#heading=h.trqab8y0kufp is implemented to check by type